infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/TolerationsProvisioner.java

```diff
@@ -33,19 +28,8 @@ public class TolerationsProvisioner implements ConfigurationProvisioner {
   private final List<Toleration> tolerations;
 
   @Inject
-  public TolerationsProvisioner(
```

Comment:
why can't we completely remove the Tolearation from che-server ? AFAIK it is now managed not on the che-server side but on the operator level cc: @tolusha

Reply:
We can remove it. It is managed by operator.

Reply:
Sounds good, I can remove it completely :+1: